code/game/atoms_movable.dm

```diff
@@ -316,7 +316,7 @@
 	TT.target_turf = get_turf(target)
 	TT.init_dir = get_dir(src, target)
 	TT.maxrange = range
-	TT.speed = speed
+	TT.speed = max(speed, 1)

```

Comment:
A speed lower then 1 is suppose to be supported. the mathing of where to place them is what should be fixed. the rounding should be made into a ceiling()